### ISSUE TITLE
请问有1D-CNN的脚本吗？自己设计的（如下所示）一直报错解决不了，在python里面搭建完显示正常，放在fate里面就报错

### ISSUE THREAD

**Issue by DuJiahao1996** [FederatedAI/FATE#3382]
这是我自己设计的：
import keras
from keras.models import Sequential
from keras.layers import Conv1D, MaxPooling1D, Reshape,Dense, Flatten
model=Sequential()
model.add(Conv1D(16, 8,  padding='same', activation='relu',input_shape=(4116,58)))
model.add(MaxPooling1D(pool_size= 3))
model.add(Flatten())
model.add(Dense(7, activation='sigmoid'))
model.summary()

网络结构输出如下
_________________________________________________________________
Layer (type) /                       Output Shape /              Param    
_________________________________________________________________
conv1d_1 (Conv1D) /           (None, 4116, 2) /           466       
_________________________________________________________________
max_pooling1d_1 /               (None, 343, 2)  /            0         
_________________________________________________________________
flatten_1 (Flatten) /                (None, 686)  /               0         
_________________________________________________________________
dense_1 (Dense) /                  (None, 7)   /                4809      
_________________________________________________________________
Total params: 5,275
Trainable params: 5,275
Non-trainable params: 0

报错如下：
[ERROR] [2021-11-17 02:38:50,747] [2648:139730708633408] - task_executor.py[line:140]: Input 0 of layer conv1d_1 is incompatible with the layer: expected ndim=3, found ndim=2. Full shape received: [None, 58]
